metrics/src/main/java/com/lightstep/tracer/metrics/CpuMetricGroup.java

```diff
@@ -0,0 +1,43 @@
+package com.lightstep.tracer.metrics;
+
+import java.io.IOException;
+
+import oshi.hardware.CentralProcessor.TickType;
+import oshi.hardware.HardwareAbstractionLayer;
+
+class CpuMetricGroup extends MetricGroup {
+  private static final CounterMetric<CpuMetricGroup,Long> cpuUsage = new CounterMetric<>("cpu.udage", Long.class);
```

Comment:
*cpu.usage

Reply:
Good eye! I added [this test](https://github.com/lightstep/lightstep-tracer-java-common/blob/cf30dfac03be1da43fc7e960a880d08f993cce65/metrics/src/test/java/com/lightstep/tracer/metrics/MetricsTest.java#L41-L47) to safeguard from these typos.